src/components/Frame/components/ContextualSaveBar/components/DiscardConfirmationModal/tests/DiscardConfirmationModal.test.tsx

```diff
@@ -7,6 +7,22 @@ import {Modal} from 'components';
 import DiscardConfirmationModal from '../DiscardConfirmationModal';
 
 describe('<DiscardConfirmationModal />', () => {
+  it('passes its open prop value to the Modal', () => {
+    const discardConfirmationModalOpen = shallowWithAppProvider(
```

Comment:
Can we use full mounting rather than shallow? In v4 we'll only have `mount`.

```suggestion
    const discardConfirmationModalOpen = mountWithAppProvider(
```

Reply:
Could we use mount rather than shallow? In `v4`, we'll only be using mount